update-center-source.properties

```diff
@@ -2,13 +2,21 @@ plugins=aemrules,ansible,apigee,authaad,authgoogleoauth,authoidc,checkstyle,clov
 scanners=scannercli,scannermsbuild,scannerant,scannermaven,scannergradle,scannerazure,scannerjenkins
 
 devVersion=10.6-SNAPSHOT
-publicVersions=9.9,9.9.1,9.9.2,9.9.3,9.9.4,10.0,10.1,10.2,10.2.1,10.3,10.4,10.4.1,10.5
+publicVersions=9.9,9.9.1,9.9.2,9.9.3,9.9.4,10.0,10.1,10.2,10.2.1,10.3,10.4,10.4.1,10.5,10.5.1
 privateVersions=4.2,4.3,4.3.1,4.3.2,4.3.3,4.4,4.4.1,4.5,4.5.1,4.5.2,4.5.4,4.5.5,4.5.6,4.5.7,5.0,5.0.1,5.1,5.1.1,5.1.2,5.2,5.3,5.4,5.5,5.6,5.6.1,5.6.2,5.6.3,5.6.4,5.6.5,5.6.6,5.6.7,6.0,6.1,6.2,6.3,6.3.1,6.4,6.5,6.6,6.7,6.7.1,6.7.2,6.7.3,6.7.4,6.7.5,6.7.6,6.7.7,7.0,7.1,7.2,7.2.1,7.3,7.4,7.5,7.6,7.7,7.8,7.9,7.9.1,7.9.2,7.9.3,7.9.4,7.9.5,7.9.6,8.0,8.1,8.2,8.3,8.3.1,8.4,8.4.1,8.4.2,8.5,8.5.1,8.6,8.6.1,8.7,8.7.1,8.8,8.9,8.9.1,8.9.2,8.9.3,8.9.4,8.9.5,8.9.6,8.9.7,8.9.8,8.9.9,8.9.10,9.0,9.0.1,9.1,9.2,9.2.1,9.2.2,9.2.3,9.2.4,9.3,9.4,9.5,9.6,9.6.1,9.7,9.7.1,9.8,
 
 ltsVersion=9.9.4
 ltaVersion=9.9.4
 pastLtaVersion=8.9.10
 
+10.5.1.description=Bug fix
+10.5.1.downloadUrl=https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-10.5.1.xxxxx.zip
```

Comment:
???

Reply:
Preparing the work for a future bug fix release, in case another PM has to handle it during I'm off.

Reply:
I thought the very next day was awfully fast :sweat_smile: